flower/utils/results/stores/__init__.py

```diff
@@ -0,0 +1,31 @@
+from celery.backends.base import BaseBackend
+
+from flower.utils.results.stores.abstract import AbstractBackendResultsStore
+
+try:
+    from celery.backends.database import DatabaseBackend
+except ImportError:
+    DatabaseBackend = None
```

Comment:
nit,
i really don't like silently ignored ImportErrors, especially if any branching is done based on the presence of an incomplete dependencies. it might not make sense to log the currently-squalched error, but i would generally prefer to see that error logged rather than squalched.

Author's descretion

Reply:
Actually, this isn't an error case. The pattern of:
```python
try:
    import something
except ImportError:
    something = None
```
is pretty common in Python. This is often how libraries handle optional dependencies. It's not immediately an error if someone can't import `DatabaseBackend`, since `DatabaseBackend` is dependent on SQLAlchemy existing.

It _is_ an error elsewhere if they're trying to use `DatabaseBackendResultsStore`, though, since that class only makes sense if you have SQLAlchemy installed. But in that case, we have no try/except guard around the import, so it will hard-fail as intended.

Reply:
shadowing the missing lib as None seems like an anti-pattern to me, if it were me i would prefer non-semipredicat'ing this and store the `HAS_SOMETHING = False; try :import something; HAS_SOMETHING=True; except ImportError as err: LOGGER.warning('something processing disabled due to %s, 'err); ... ; if not HAS_SOMETHING: raise (RuntimeError('cannot be configured to use Something when something cannot be imported'), import something)` etc.

shippit